pkgs/applications/audio/exaile/default.nix

```diff
@@ -51,13 +51,14 @@ stdenv.mkDerivation rec {
     gst-plugins-ugly
     gst-libav
   ]) ++ (with python3.pkgs; [
-    bsddb3
     dbus-python
     mutagen
     pygobject3
     pycairo
     gst-python
-  ]) ++ lib.optional deviceDetectionSupport udisks
+  ] ++ lib.optional (pythonAtLeast "3.10") berkeleydb
+  ++ lib.optional (pythonOlder "3.9") bsddb3)
```

Comment:
`pythonOlder` is `<` not `<=`, so if you had Python 3.9, you wouldn't have either of the two packages.
It should probably be `pythonOlder "3.10"`